static/extensions/dumzdev/removebg.js

```diff
@@ -0,0 +1,95 @@
+// Name: Background Remover
+// ID: backgroundRemover
+// Description: Removes background from images. 
+// By: dumzdev, Kaleido AI  <https://scratch.mit.edu/users/dumzdev/>
+// License: MIT
+
+class BackgroundRemoverExtension {
+    constructor() {
+        this.apiKey = '4LjLg3UzLPWtkU2VX9H38hqx'; // Default API Key
+    }
+
+    getInfo() {
+        return {
+            id: 'backgroundRemover',
+            name: 'Background Remover',
+            blocks: [
+                {
+                    opcode: 'setApiKey',
+                    blockType: Scratch.BlockType.COMMAND,
+                    text: 'set API key to [APIKEY]',
+                    arguments: {
+                        APIKEY: {
+                            type: Scratch.ArgumentType.STRING,
+                            defaultValue: '4LjLg3UzLPWtkU2VX9H38hqx'
+                        }
+                    }
+                },
+                {
+                    opcode: 'removeBackground',
+                    blockType: Scratch.BlockType.REPORTER,
+                    text: 'remove background from image [IMAGE]',
+                    arguments: {
+                        IMAGE: {
+                            type: Scratch.ArgumentType.STRING,
+                            defaultValue: 'data:image/png;base64,...'
+                        }
+                    }
+                }
+            ],
+            menus: { },
+            color1: '#FFC536', // Main theme color
+            color2: '#FFD06B', // Lighter shade of #FFC536
+            color3: '#FFC536',
+            menuIconURI: 'https://files.catbox.moe/ezlx70.svg', // Menu icon
+            blockIconURI: 'https://files.catbox.moe/ezlx70.svg', // Block icon
+            textColor: '#000000', // Black text on the blocks
+            docsURI: 'https://www.remove.bg/api' // Button linking to Remove.bg API
```

Comment:
These should be before the blocks, not after.

Also the block and menu icons should be a data.uri not a direct url.

Reply:
now the data uri thing is a hill I will die on over this cause needing to paste the raw image data into a JSON argument is terrible

Reply:
done :)